content/kubermatic/master/tutorials_howtos/networking/ipam/_index.en.md

```diff
@@ -0,0 +1,157 @@
++++
+title = "Multi-Cluster IP Address Management (IPAM)"
+date = 2022-07-26T14:45:00+02:00
++++
+
+Feature responsible for automating the allocation of ranges of IP addresses per-user-cluster, based on a predefined configuration ([IPAMPool](#input-resource-ipampool)) per datacenter that defines the pool subnet and the allocation size. The user cluster allocated ranges are available in the [KKP Addon](#kkp-addon-template-integration) `TemplateData`, so it can be used by various applications / Addons running in the user cluster.
+
+{{< table_of_contents >}}
+
+## Motivation and Background
+Networking applications deployed in KKP user clusters need automated IP Address Management (IPAM) for IP ranges that they use, in a way that prevents address overlaps between multiple user clusters. An example for such an application is MetalLB load-balancer, for which a unique IP range from a larger CIDR range needs to be configured in each user cluster in the same datacenter. This is currently mostly done manually, by keeping evidence of allocated subnets in an external document and copy-pasting from it when deploying the MetalLB addon.
```

Comment:
I would rather skip the last sentence " This is currently mostly done manually, by keeping evidence of allocated subnets in an external document and copy-pasting from it when deploying the MetalLB addon."